src/core/thread/mle.cpp

```diff
@@ -2852,11 +2848,12 @@ otError Mle::HandleChildUpdateRequest(const Message &aMessage, const Ip6::Messag
     uint8_t tlvs[kMaxResponseTlvs] = {};
     uint8_t numTlvs = 0;
 
-    otLogInfoMle(GetInstance(), "Received Child Update Request from parent");
-
     // Source Address
     SuccessOrExit(error = Tlv::GetTlv(aMessage, Tlv::kSourceAddress, sizeof(sourceAddress), sourceAddress));
     VerifyOrExit(sourceAddress.IsValid(), error = OT_ERROR_PARSE);
+
+    LogMleMessage("Receive Child Update Request from parent", aMessageInfo.GetPeerAddr(), sourceAddress.GetRloc16());
```

Comment:
Suggestion: what do you think of "Received" instead of "Receive" here?

Reply:
No strong preference.  It depends on whether you view this as:
1. The packet is ***received*** from the lower layer.
2. The packet ***receive*** handler is executing.

Reply:
I personally like past tense for "Received",  just tto be consistent with mesh-forwarder/mac logs. 

For outgoing message I like  "Send <something>" or "Sending <something>" if we are queuing the frame...and "Sent <something>" if it's already sent...

Reply:
The difference is that the logs in MLE are typically printed at the beginning of the MLE handler with lots of MLE-related logic that will follow, whereas MAC and MeshForwarder (in most cases) typically print the logs once they are done processing the message.

Reply:
That makes sense.... they are printed before the message is parsed...let's keep it "Receive"...